lib/diagnostic.rb

```diff
@@ -43,6 +43,7 @@ def transform(words)
   # replacing the trailing newline character with a
   # space and return the result.
   def read_file(filename)
-    # Your code goes here
+    # file = File.new(filename)
+    # file.reduce {|memo, line| memo = line.sub(\n, )}
```

Comment:
Here's the official solution for this one.
`File.open(filename).reduce('') { |a, e| a << e.chomp + ' ' }.strip`